modules/store-devtools/src/instrument.ts

```diff
@@ -60,7 +60,7 @@ export function createConfig(
     actionSanitizer: undefined,
     stateSanitizer: undefined,
     name: DEFAULT_NAME,
-    serialize: false,
+    serialize: undefined,
```

Comment:
This potentially would be a breaking change, since the existing behavior is to handle circular references. We should have changed with the default serializer for router-store.

Reply:
As far as I can tell, `undefined` is [recommended default](https://github.com/zalmoxisus/redux-devtools-extension/blob/master/docs/API/Arguments.md#serialize) (will use regular JSON.stringify to send data, and it's the fasted mode).
There are concerns that attempting to resolve circular references with [JSPB](https://developers.google.com/protocol-buffers/docs/reference/javascript-generated) might crash the extension.

Reply:
👍